src/main/java/io/excaliburfrc/robot/Constants.java

```diff
@@ -10,6 +10,12 @@
  */
 public final class Constants {
   // add a inner class `public static final class` for each subsystem.
+  public static final class IntakeConstants {
+    public static final int INTAKE_MOTOR_ID = 0;
+    public static final int FORWARD_CHANNEL = 0;
+    public static final int REVERSE_CHANNEL = 1;
```

Comment:
Real ports are: 
```
motor id = 60
fwd = 1
rev =  0
``` 
Don't change the constant names though - only the port numbers.